pandas/core/internals/managers.py

```diff
@@ -1275,7 +1278,8 @@ def grouped_reduce(self: T, func: Callable, ignore_failures: bool = False) -> T:
         else:
             index = Index(range(result_blocks[0].values.shape[-1]))
 
-        if ignore_failures:
```

Comment:
can you kill ignore_failures now?

Reply:
No, though we can get rid of it in 2.0 BC of the nuisance column derecations